src/urllib3/util/timeout.py

```diff
@@ -132,15 +142,15 @@ def _validate_timeout(cls, value, name):
                 "be an int, float or None."
             )
         try:
-            float(value)
+            float(value)  # type: ignore
```

Comment:
Let's add an `value = float(value)` here so we don't need to do the `# type: ignore` below?

Reply:
if we change it to `value = float(value)`, we have to keep `# type: ignore` for this line. otherwise, we will get:

```src/urllib3/util/timeout.py:145: error: Argument 1 to "float" has incompatible type "object"; expected "Union[SupportsFloat, _SupportsIndex, str]"```

and [test_timeout_str](https://github.com/urllib3/urllib3/blob/26e9f27692193f898e78f88ea8169558ced30d12/test/test_util.py#L597) will fail because the numbers are int now.
But, Yes, we don't need below `# type: ignore` after that.

Should we change it?